examples/vue_js_tracker/README.md

```diff
@@ -0,0 +1,71 @@
+## vue_js_tracker
+
+This is an example Vue.js application to demonstrate how to install the the GDX_Analytics Snowplow Standalone Tracker vA2.10.2 in the Vue.js framework.
+
+## Features
+
+This app contains the Snowplow Tracker Standalone va2.10.2, which is installed in the header
+section of index.html. It also leverages the Vue Router Add-on by calling the Snowplow 'trackpageview' function, whenever the Vue Router changes the navigation.
+
+
+## Project Status 
+
+This project is currently under development and actively supported by the GDX Analytics Team.
+
+## Requirements
+
+This project requires Vue CLI and npm. Vue CLI requires Node.js version 8.9 or above. 
+
+### Installation instructions
+
+To install Vue CLI (globally)
+
+```
+npm install -g @vue/cli
+```
+
+See the official Vue.js installation documentation here: https://cli.vuejs.org/guide/installation.html
+
+## To run this application
+
+From the root of the application, run the following commands to install dependencies and serve the application using the vue-cli-service. This will serve the application from localhost.
+
+
+### Install the project dependencies in the node_modules folder
+
+```
+npm install
```

Comment:
This is fine, but also add the alternative approach to installing in a way that respects the pinned package-lock.json.
```
npm ci
```

Reply:
Ive updated the readme to use npm ci instead.